### ISSUE TITLE
Proxy url not being used

### ISSUE THREAD

**Issue by kavinderd** [tonycoco/heroku-buildpack-ember-cli#62]
I have setup my ember heroku app with an `API_URL` and `API_PROXY_PATH` as demonstrated in the readme, but my backend api is not receiving any request.  

Is there any configuration I need to do aside from setting the heroku configs?

**Comment by tonycoco** [comment]
May be a duplicate of #60... looking into this.

**Comment by tonycoco** [comment]
One thing you could do is tail the logs for each Heroku server and make sure the requests are routing the way you are expecting them to. You should take a look at this issue as well, #66

**Comment by kavinderd** [comment]
Sorry for the late reply, busy with another project. The issue I'm seeing is similar to #66 except in my logs for my ember application I don't even see requests being routed to my backend server.
```
API_URL=https://my-backend.herokuapp.com/
API_PROXY_PATH=/api/
```
I have a route `/courses` that should be routing to `https://my-backend.herokuapp.com/` instead in the logs I see
```
2015-03-28T22:16:29.364257+00:00 heroku[router]: at=info method=GET path="/courses" host=ember-app.herokuapp.com ...
```
And like in #66 's case I have proxied my local ember app to the remote backend and it works as expected.

**Comment by erichonkanen** [comment]
+1 I have 2 heroku setups each with a backend app and client app... one of them I created almost a year ago and it works, the 2nd one I created tonight and using same set up but the proxy isnt going through for some reason

**Comment by erichonkanen** [comment]
I got proxy working by adding a Procfile with:

`web: ember serve --port $PORT --live-reload=false --proxy http://craft-backend-production.herokuapp.com`

**Comment by tonycoco** [comment]
@erichonkanen this is not recommended, since now you are basically running a node server for a static site and bypassing the nginx setup completely.

**Comment by mattmcginnis** [comment]
@tonycoco , if  @erichonkanen 's solution isn't ideal, what is?  This issue still exists and I'm running into it as well. ( https://github.com/tonycoco/heroku-buildpack-ember-cli/issues/114 )@kavinderd , did you find a solution?

What I noticed is that the code below assumes your API_PREFIX_PATH to be /api/

```javascript
    <% if ENV["API_URL"] %>
      location <%= ENV["API_PREFIX_PATH"] || "/api/" %> {
        proxy_pass <%= ENV["API_URL"] %>;
        proxy_set_header Real-IP $remote_addr;
        proxy_set_header Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header NginX-Proxy true;
        proxy_ssl_session_reuse off;
        proxy_redirect off;
        <% if ENV["NGINX_DEBUG"] %>add_header Ember-Cli-Proxy on;<% end %>
      }
    <% end %>
```

My API server does not use a prefix path and setting the prefix path to "" or "/" does not solve this issue.  

In any case, I scoped all of my rails controllers under /api/ and tried using the prefix path and it still didn't work.  All requests go to myemberclidomain.com instead of myapiserver.com.  My api server does not show any request in the logs and all API calls return a 405 because myemberclidomain.com is receiving the request instead of the myapiserver.com.

Any ideas? Is the current version of this repo working for anyone using a proxy?
